src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Storage storage) {
```

Comment:
According to the encapsulation principle, it's a good practice to hide the inner workings of classes. Here, the `handle` method directly accesses the `Storage` class. This could be improved by using a service layer that interacts with the `Storage` and is then used by the `BalanceOperation`.